CorsixTH/Lua/persistance.lua

```diff
@@ -63,7 +63,7 @@ function unpermanent(name)
   saved_permanents[name] = nil
 end
 
-local --[[persistable:persistance_global_fetch]] function global_fetch(...)
+local --[[persistable:persistence_global_fetch]] function global_fetch(...)
```

Comment:
I agree that the spelling is indeed unfortunate (even the file is spelt wrong). But there is a greater impact for amending a persistable function's name. I would hazard we don't change it unless we really have to.

Reply:
Indeed, that name is used in old savegames, they will break if you change the name.

Reply:
Thanks! Reverted this.